README.md

```diff
@@ -1 +1,12 @@
-# viveport_sdk_unreal_sample
\ No newline at end of file
+# VIVEPORT SDK Unreal Sample
+
+## For DRM wrapper Error, other use this VIVEPORT SDK DRM solution.
+
+* Download ViveportSDK Unreal. https://developer.viveport.com/documents/sdk/en/unrealengine.html
+* Download DRM_fix_Component.
+* Copy DRM_fix_Component\ViveportSDK folder to your Unreal project .\Plugins folder.
+* Change .\Plugins\ViveportSDK\Source\ViveportSDK\Classes\ViveportDRMComponent.h to your VIVEPORT ID and VIVEPORT KEY.
+* Remove .\Plugins\ViveportSDK\Binaries , .\Plugins\ViveportSDK\Intermediate , your Project's .\Binaries and your Project's .\Intermediate folder.
+* Start your .uproject. ureanl will start rebuild your project.
+* Create new Actor in your map and add Viveport DRM component . 
+* Run your project. If your viveport account doesn't have this app, Unral editor will end. This VIVEPORT SDK DRM is work.
```

Comment:
Run your project. If your VIVEPORT account doesn't own this app, Unreal editor will close. It means VIVEPORT DRM SDK is workable now.